pkg/crc/network/proxy.go

```diff
@@ -79,8 +81,9 @@ func (p *ProxyConfig) HttpsProxyForDisplay() string {
 }
 
 // AddNoProxy appends the specified host to the list of no proxied hosts.
-func (p *ProxyConfig) AddNoProxy(host string) {
-	p.NoProxy = fmt.Sprintf("%s,%s", host, p.NoProxy)
+func (p *ProxyConfig) AddNoProxy(hosts string) {
```

Comment:
>  you want to be able to pass ".example.com,.example2.com"

yes, that was the intention, I should change the signature for this to `host ...string` so it will become a variadic method.